src/z_zone.h

```diff
@@ -69,5 +70,14 @@ unsigned int Z_ZoneSize(void);
 #define Z_ChangeTag(p,t)                                       \
     Z_ChangeTag2((p), (t), __FILE__, __LINE__)
 
+static inline char* Z_StringCopy(const char *src, int tag)
```

Comment:
The function name is confusing because it's more equivalent to `M_StringDuplicate` in my opinion.

But I'm not convinced we should be using the zone heap to allocate strings anyway. That's really for larger allocations like WAD lumps and things that need to be efficiently allocated and cached while the game is running. Should we just switch to using the normal C heap for the dehacked string table?